spec/unit/hanami/assets/sass/engine_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'hanami/assets/sass/engine'
+
+RSpec.describe Hanami::Assets::Sass::Engine do
+  let(:template) { double('template') }
+  let(:source) { ::SassC::Engine }
+  let(:source_engine) { instance_double(::SassC::Engine) }
+
+  it 'wraps the source Sass renderer' do
+    expect(source).to receive(:new).with(template, {})
+    described_class.new(template)
+  end
+
+  describe '#render' do
+    it 'forwards the call to the wrapped source' do
+      allow(source).to receive(:new).and_return(source_engine)
```

Comment:
@jodosha I wanted to assert that the outgoing message was sent. Since we don't have control over _which_ instance of `SassC::Engine` I opted for this approach. I'm totally open to changing it. What would you like to see here?

Reply:
@landongrindheim In the shed of the light of my last [review](https://github.com/hanami/assets/pull/108#pullrequestreview-287246316), this doesn't matter anymore.